data_structures/src/superblock.rs

```diff
@@ -509,9 +546,9 @@ mod tests {
         );
 
         // The only think that should change is the superblock_index
-        expected_sbs.current_superblock_index = Some(1);
-        // And the superblock_hash, which will be set to None
-        expected_sbs.current_superblock_hash = None;
+        expected_sbs.current_superblock_index = 1;
+        // And the superblock_hash, which will be set to the previous superblock
+        expected_sbs.current_superblock_hash = _sb1.hash();
```

Comment:
You should remove the `_` if the variable is no longer unused.

```suggestion
        expected_sbs.current_superblock_hash = sb1.hash();
```